### PR TITLE
IO1-3028: changed cluster format to from 0x1234...1235 to 1234....1244

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const ValidatorSuccessScreen = () => {
           <Grid item container className={classes.Wrapper}>
             <Grid item className={classes.Text}>Your new validator is managed by the following cluster:</Grid>
             <Grid container item className={classes.ClusterID}>
-              <Typography>Validator Cluster {longStringShorten(clusterHash, 4)}</Typography>
+              <Typography>Validator Cluster {longStringShorten(clusterHash, 4).slice(2)}</Typography>
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx:54]: It is better to use more deterministic way of cutting first `0x` for the cases when there is no leading `0x`.
Correct way: `.replace(/^0x/, '');`

**Review by meshin-blox — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx (hunk 1 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx:54]: done

File: src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ const ValidatorSuccessScreen = () => {
           <Grid item container className={classes.Wrapper}>
             <Grid item className={classes.Text}>Your new validator is managed by the following cluster:</Grid>
             <Grid container item className={classes.ClusterID}>
-              <Typography>Validator Cluster {longStringShorten(clusterHash, 4)}</Typography>
+              <Typography>Validator Cluster {longStringShorten(clusterHash, 4).replace(/^0x/, '')}</Typography>
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx:54]: It was not fixed according to new format of shorten function

**Review by meshin-blox — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx:54]: @meshin-blox 
https://github.com/bloxapp/ssv-web/pull/136
please merge it before, I make the pull from stage and fix it

Comment (axelrod-blox) [reply src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx:54]: done

**Review by meshin-blox — APPROVED** [review]

**Pull request merged**